charts/prometheus-blackbox-exporter/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Prometheus Blackbox Exporter
 name: prometheus-blackbox-exporter
-version: 4.5.2
+version: 4.5.3
```

Comment:
new feature should raise minor version to 4.6.0.

Reply:
Bumped version to 4.6.0.